canvashare/canvashare.py

```diff
@@ -130,13 +132,13 @@ def update_drawing_info(artist_name, drawing_id):
     # If request is for viewing the drawing, increase view count without
     # requiring user to be logged in
     if data['request'] == 'view':
-        with open('canvashare/drawing_info/' + artist_id + '/' + drawing_id
```

Comment:
I think this may not work, did you test this endpoint?

Reply:
Thanks for catching that! I just amended the commit to correct that.